### DISCUSSION TITLE
Is there a glyph that can be used in a similar manner to nf-ple-lower_left_triangle (etc), but is a curve rather than a triangle?

### DISCUSSION THREAD

**Discussion by iaingalloway** [ryanoasis/nerd-fonts#1152] [Q&A]
I've been searching for a set of four characters that I can use to draw filled quarter-circles (or quarter-ellipses). There are plenty of triangles and half-circles, but I couldn't find any quarter-circles.

If there isn't one, what would be the best place to start creating one to add?

**Comment by Finii** [comment]
These really quarter cicles, but rounded corners?

![image](https://user-images.githubusercontent.com/16012374/227267483-8a8c21cc-68f3-431b-b776-3a6d33c0ed61.png)

But these are outlines and can not be used to draw filled quarter things.

Then there is the set of quarter blocks, which are there in any combination.

![image](https://user-images.githubusercontent.com/16012374/227268134-0001321e-c8a8-41a5-9b16-20b4290868c2.png)

(They look distorted here, but are really 1/4 chunks of the 'cell'.)

> what would be the best place to start creating one to add?

I would just fire up ~~`Glyphs`~~ `Inkscape` and draw them, from 3 points.

![image](https://user-images.githubusercontent.com/16012374/227270051-77cf1b26-2e28-4597-a981-9ce6897077e8.png)

**Comment by iaingalloway** [comment]
Yeah, I can create the actual glyphs (and patch the font locally).

I more meant, if I wanted to contribute the new glyphs, where's the best place to do that? Directly here, or in one of the icon libraries NF pulls from?

I presume the best place would be https://github.com/ryanoasis/powerline-extra-symbols but I'm not sure.
